server/middleware/auditMiddleware.ts

```diff
@@ -41,14 +80,19 @@ const pageViewEventMap: Record<string, Page> = {
   '/prisoners/:prisonNumber/induction/qualification-level': Page.INDUCTION_UPDATE_QUALIFICATION_LEVEL,
   '/prisoners/:prisonNumber/induction/qualification-details': Page.INDUCTION_UPDATE_QUALIFICATION_DETAILS,
   '/prisoners/:prisonNumber/induction/additional-training': Page.INDUCTION_UPDATE_ADDITIONAL_TRAINING,
+  '/prisoners/:prisonNumber/induction/check-your-answers': Page.INDUCTION_UPDATE_CHECK_YOUR_ANSWERS,
 
-  '/prisoner/:prisonNumber/work-and-skills/in-prison-courses-and-qualifications':
-    Page.IN_PRISON_COURSES_AND_QUALIFICATIONS,
+  // Non audit routes. These routes do not raise an audit event
+  '/plan/:prisonNumber/induction-created': null,
```

Comment:
This route is just used in a redirect, so no need to raise an audit event for it.